app/src/main/java/de/tum/in/tumcampusapp/services/JobIdConstants.java

```diff
@@ -0,0 +1,16 @@
+package de.tum.in.tumcampusapp.services;
+
+/**
+ * Created by ansel on 11/11/2017.
+ */
+
+final class JobIdConstants {
```

Comment:
Sorry, I wasn't too clear on this. I meant this class: https://github.com/TCA-Team/TumCampusApp/blob/master/app/src/main/java/de/tum/in/tumcampusapp/auxiliary/Const.kt

Just put any new constants there.